aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/layout/AlfStackContainer.get.js

```diff
@@ -0,0 +1,324 @@
+               },
+               {
+                  name: "alfresco/layout/VerticalWidgets",
+                  config: {
+                     widgets: [
+                        {
+                           name: "alfresco/buttons/AlfButton",
+                           config: {
+                              id: "DELETE_PANE_1",
+                              label: "Delete pane 5 (idx 4)",
+                              publishTopic: "STACKCONTAINER_DELETE_PANE_TOPIC",
+                              publishPayload: {
+                                 index: 4
+                              }
+                           }
+                        },
+                        {
+                           name: "alfresco/buttons/AlfButton",
+                           config: {
+                              id: "DELETE_PANE_2",
+                              label: "Delete pane titled 'logo3'",
+                              publishTopic: "STACKCONTAINER_DELETE_PANE_TOPIC",
+                              publishPayload: {
+                                 title: "logo4"
+                              }
+                           }
+                        },
+                        {
+                           name: "alfresco/buttons/AlfButton",
+                           config: {
+                              id: "DELETE_PANE_3",
+                              label: "Delete pane with id 'dijit_layout_ContentPane_2'",
+                              publishTopic: "STACKCONTAINER_DELETE_PANE_TOPIC",
+                              publishPayload: {
+                                 id: "dijit_layout_ContentPane_2"
+                              }
+                           }
+                        }
+                     ]
+                  }
+               },
+               {
+                  name: "alfresco/layout/VerticalWidgets",
+                  config: {
+                     widgets: [
+                        {
+                           name: "alfresco/buttons/AlfButton",
+                           config: {
+                              id: "ADD_PANE_99",
+                              label: "Add pane 99",
+                              publishTopic: "STACKCONTAINER_ADD_PANE_TOPIC",
+                              publishPayload: {
+                                 widgets: [
+                                    {
+                                       name: "alfresco/html/Label",
+                                       title: "logo99",
+                                       selected: true,
+                                       config: {
+                                          label: "logo99"
+                                       }
+                                    }
+                                 ]
+                              }
+                           }
+                        }
+                     ]
+                  }
+               }
+            ]
+         }
+      }
+   ]
```

Comment:
Why not raise a feature request for that and we'll prioritize it on the backlog... in the meantime, why not just add the 3 lines of code (of which 2 lines are just brackets) that we're politely asking for.